Tools/CISupport/ews-build/steps.py

```diff
@@ -425,8 +431,8 @@ def evaluateCommand(self, cmd):
             if identifier:
                 identifier = identifier.replace('master', 'main')
             self.setProperty('identifier', identifier)
-            ews_revision = self.getProperty('ews_revision')
-            if ews_revision:
+            revision = self.getProperty('ews_revision') or self.getProperty('github.base.sha')
```

Comment:
This particular change doesn't look right (and I believe should be removed). ews_revision is set when a build is triggered by other build (e.g.: a wk2 queue) (or when using force build in test mode).

If ews_revision is not present, it would mean CheckOutSpecificRevision step was not run, so we shouldn't try addURL to CheckOutSpecificRevision step.

Separately, I guess we might not even need CheckOutSpecificRevision step in PR workflow. We need it in patch workflow to ensure that builder and tester are using same revision (builders pass on the revision they used to the tester in ews_revision property which testers checkout in CheckOutSpecificRevision step). In PR workflow, both builder and testers can simply checkout same branch.

Reply:
Thanks for explaining this, that makes sense. I will remove this part for now, we'll need to figure out triggering builds in the near future, but that seems out of scope of this change